### ISSUE TITLE
Build failure: php8.3

### ISSUE THREAD

**Issue by nklmilojevic** [NixOS/nixpkgs#345331]
### Steps To Reproduce

1. build php8.3

### Build log

```
error: builder for '/nix/store/al6qib6cyajff2383xqvpqw11ksqmr61-php-soap-8.3.12.drv' failed with exit code 2;
       last 10 log lines:
       > ---------------------------------------------------------------------
       > Time taken      :     4 seconds
       > =====================================================================
       >
       > =====================================================================
       > FAILED TEST SUMMARY
       > ---------------------------------------------------------------------
       > SOAP Bug #71610 - Type Confusion Vulnerability - SOAP / make_http_soap_request() [tests/bug71610.phpt]
       > =====================================================================
       > make: *** [Makefile:136: test] Error 1
       For full logs, run 'nix log /nix/store/al6qib6cyajff2383xqvpqw11ksqmr61-php-soap-8.3.12.drv'.
```

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.0.0, macOS 15.0`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.8`
 - nixpkgs: `/nix/store/dgxr8vqspz6cncwy1pqpppsn9abhfn9x-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by Necriso** [comment]
when will this be fixed?
Nothing is working at the moment.

**Comment by shyim** [comment]
I tried to reproduce, but it works for me with latest nixpkgs 🤔

**Comment by nklmilojevic** [comment]
I have just checked unstable and this is indeed fixed with PHP 8.3.

**Comment by shyim** [comment]
then let's close this right?
